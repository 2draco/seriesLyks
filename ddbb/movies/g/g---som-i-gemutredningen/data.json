{"rating": 5, "links": [], "maingenre": null, "imdb": "tt0249528", "year": "1985", "plot": [""], "genres": [], "compose": [], "title": "G - som i gemutredningen", "plot_es": [""], "original_name": "G - som i gemutredningen", "mediaType": 2, "languages": ["Swedish"], "idm": 1112383, "plot_en": [""], "status": 3, "timestamp": 1419289200, "broadcast": "online", "produce": [{"imdb": "0079546", "name": "Sveriges Television (SVT)"}], "total_users": 0, "id_media": "Z4UF5KPADPY6RE7", "director": [{"name": "Jonas Frick"}], "country": ["Sweden"], "cast": [{"role": "Ivar", "name": "Staffan Ling"}, {"role": "-", "name": "Gunnel Fred"}], "write": [], "episodeUserCreator": [], "aka": null}