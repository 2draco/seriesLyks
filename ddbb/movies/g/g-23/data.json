{"rating": 5, "links": [], "maingenre": "Short", "imdb": "tt0479041", "year": "2005", "plot": [""], "genres": ["Short"], "compose": [], "title": "G-23", "plot_es": [""], "original_name": "G-23", "mediaType": 2, "languages": ["Mandarin"], "idm": 1166268, "plot_en": [""], "status": 3, "timestamp": 1419289200, "broadcast": "online", "produce": [], "total_users": 0, "id_media": "P9UTSA4UHTXV7DK", "director": [{"name": "Anthony Chen"}], "country": ["Singapore"], "cast": [{"role": "Indian Teenager", "name": "Joane Jonathan"}, {"role": "-", "name": "Rick Tan"}, {"role": "-", "name": "Li Xie"}], "write": [{"name": "Anthony Chen"}], "episodeUserCreator": [], "aka": null}