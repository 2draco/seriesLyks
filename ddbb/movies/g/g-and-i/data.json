{"rating": 5, "links": [], "maingenre": "Short", "imdb": "tt2262452", "year": "2012", "plot": ["A young girl creates an imaginary friend to help her escape everyday life."], "genres": ["Short", "Adventure", "Drama", "Fantasy"], "compose": [], "title": "G and I", "plot_es": ["A young girl creates an imaginary friend to help her escape everyday life."], "original_name": "G and I", "mediaType": 2, "languages": ["English"], "idm": 1170321, "plot_en": ["A young girl creates an imaginary friend to help her escape everyday life."], "status": 3, "timestamp": 1419289200, "broadcast": "online", "produce": [], "total_users": 0, "id_media": "9WWHEU7H7Y7VVFN", "director": [{"name": "Adrian Guerrero"}], "country": ["USA"], "cast": [{"role": "Griz", "name": "Karla Gastel"}, {"role": "G", "name": "Ariana Guerra"}, {"role": "Martha", "name": "Katherine Gastel"}, {"role": "Esteban", "name": "Horacio Ramirez"}], "write": [{"name": "Adrian Guerrero"}, {"name": "Adrian Guerrero"}], "episodeUserCreator": [], "aka": null, "runtime": "18"}