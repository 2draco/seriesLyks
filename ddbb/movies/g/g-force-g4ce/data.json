{"rating": 8, "links": [], "ratings_num": 1, "maingenre": "", "imdb": "tt0793844", "year": "2001", "plot": [""], "genres": ["", ""], "compose": [], "title": "\"G Force\" G4ce", "plot_es": [""], "original_name": "\"G Force\" G4ce", "mediaType": 2, "write": [], "plot_en": [""], "status": 3, "languages": [], "timestamp": 1419289200, "broadcast": "online", "produce": [], "total_users": 7, "id_media": "6V7KSU9XHN99PDP", "director": [], "rate_summary": {"aggregate": {"1": 0, "3": 0, "2": 0, "5": 0, "4": 1}}, "country": [], "cast": [], "idm": 619746, "episodeUserCreator": [], "aka": null}