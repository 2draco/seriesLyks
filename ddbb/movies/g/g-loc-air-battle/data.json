{"rating": 5, "links": [], "maingenre": "Action", "imdb": "tt1977800", "year": "1990", "plot": [""], "genres": ["Action"], "compose": [], "title": "G-Loc: Air Battle", "plot_es": [""], "original_name": "G-Loc: Air Battle", "mediaType": 2, "languages": ["English"], "idm": 1128941, "plot_en": [""], "status": 3, "timestamp": 1419289200, "broadcast": "online", "produce": [{"imdb": "0033982", "name": "SEGA/AM2 Company"}], "total_users": 0, "id_media": "TRZ6542ZUE4699D", "director": [], "country": ["Japan"], "cast": [], "write": [], "episodeUserCreator": [], "aka": null}