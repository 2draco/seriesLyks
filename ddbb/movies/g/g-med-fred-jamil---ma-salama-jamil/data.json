{"rating": 9, "links": [{"lang": "VO", "info": "Parte 1", "linksType": "streaming", "added": "2014-07-14 22:24:01", "sprite": "youtube", "idv": 9395501, "NickAddedBy": "Rosauron", "price": "", "addedBy": 124758, "url": "http://youtube.com/watch?v=eG6jZVdCdlw", "host": "Youtube", "DateAdded": "2014-07-14", "features": "720p", "quality": "Alta", "UploadedBy": "Rosauron", "sub": "Ingl\u00e9s"}], "ratings_num": 1, "maingenre": "Action", "imdb": "tt0431842", "year": "2008", "plot": ["An Arabic tale that takes place in Copenhagen, Denmark. About ancient religious hatred, about love, punishment, guilt and redemption, about being responsible for one's own actions and ..."], "genres": ["Action", "Drama"], "compose": [], "title": "G\u00e5 med fred Jamil - Ma salama Jamil", "plot_es": ["An Arabic tale that takes place in Copenhagen, Denmark. About ancient religious hatred, about love, punishment, guilt and redemption, about being responsible for one's own actions and ..."], "original_name": "G\u00e5 med fred Jamil - Ma salama Jamil", "mediaType": 2, "languages": ["Danish", "Arabic"], "idm": 1104178, "plot_en": ["An Arabic tale that takes place in Copenhagen, Denmark. About ancient religious hatred, about love, punishment, guilt and redemption, about being responsible for one's own actions and ..."], "status": 3, "timestamp": 1419289200, "broadcast": "online", "produce": [{"imdb": "0001935", "name": "Film i V\u00e4st"}, {"imdb": "0182270", "name": "New Danish Screen"}, {"imdb": "0113606", "name": "Nordisk Film- & TV-Fond"}], "total_users": 0, "aka": null, "director": [{"name": "Omar Shargawi"}], "rate_summary": {"aggregate": {"1": 0, "3": 0, "2": 0, "5": 1, "4": 0}}, "country": ["Denmark"], "cast": [{"role": "Adam", "name": "Elias Samir Al-Sobehi"}, {"role": "Mahmoud", "name": "Khalid Alssubeihi"}, {"role": "Salah", "name": "Salah El Koussa"}, {"role": "Abu Adeep", "name": "Diab El Masri"}, {"role": "The Egyptian", "name": "Hassan El Sayed"}, {"role": "Husam", "name": "Fouad Ghazali"}, {"role": "Abu Moussa", "name": "Nehmedo Ibrahim"}, {"role": "Yasmina", "name": "Amira Helene Larsen"}, {"role": "Jamil", "name": "Dar Salim"}, {"role": "Abu Jamil", "name": "Munir Shargawi"}], "write": [{"name": "Mogens Rukov"}, {"name": "Omar Shargawi"}], "episodeUserCreator": [], "id_media": "K7Y5VEHDYCS5NAW", "runtime": "90"}