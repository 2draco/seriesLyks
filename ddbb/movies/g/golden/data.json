{"rating": 5, "links": [], "maingenre": "Short", "imdb": "tt0368753", "year": "2002", "plot": [""], "genres": ["Short"], "compose": [], "title": "Golden", "plot_es": [""], "original_name": "Golden", "mediaType": 2, "languages": ["English"], "idm": 1234113, "plot_en": [""], "status": 3, "timestamp": 1419634800, "broadcast": "online", "produce": [{"imdb": "0100878", "name": "ein-blick productions"}], "total_users": 0, "id_media": "AVZ2REPTA7XCDDX", "director": [{"name": "Luci Westphal"}], "country": ["USA"], "cast": [{"role": "Abraham", "name": "Charles Leader"}], "write": [{"name": "Luci Westphal"}], "episodeUserCreator": [], "aka": null}